blueprints/BLUEPRINT_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+# *What* is being asked for?
+<!-- directions --->
+
+# *Why* is this necessary?
```

Comment:
can you change this to "why is this needed? Are there any ways currently to accomplish this?